sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/RemoteDependencyData.cs

```diff
@@ -50,7 +53,15 @@ public RemoteDependencyData(int version, Activity activity, ref ActivityTagsProc
                 case OperationType.Db:
                     var depDataAndType = AzMonList.GetTagValues(ref activityTagsProcessor.MappedTags, SemanticConventions.AttributeDbStatement, SemanticConventions.AttributeDbSystem);
                     Data = depDataAndType[0]?.ToString().Truncate(SchemaConstants.RemoteDependencyData_Data_MaxLength);
-                    Target = activityTagsProcessor.MappedTags.GetDbDependencyTarget().Truncate(SchemaConstants.RemoteDependencyData_Target_MaxLength);
+                    var dbNameAndTarget = activityTagsProcessor.MappedTags.GetDbDependencyTargetAndName();
+                    Target = dbNameAndTarget.DbTarget.Truncate(SchemaConstants.RemoteDependencyData_Target_MaxLength);
```

Comment:
nit: dbNameAndTarget.DbTarget?.Truncate

same for next line.

Reply:
truncate as a check for null so we are good.